docs/monitoring/understanding-dashboards.md

```diff
@@ -0,0 +1,460 @@
+---
+title: Understanding Geth's dashboard
+description: How to use a dashboard to understand a Geth node's performance
+---
+
+Our [dashboards page](/docs/monitoring/dashboards.md) explains how to set up a Grafana dashboard for monitoring your Geth node. This page explores the dashboard itself, explaining what the various metrics are and what they mean for the health of a node. Note that the raw data informing the dashboard can be viewed in JSON format in the browser by navigating to the ip address and port passed to `--metrics.addr` and `--metrics.port` (`127.0.0.1:6060` by default).
+
+## What does the dashboard look like?
+
+The default Grafana dashboard looks as follows (note that there are many more panels on the actual page than in the snapshot below):
+
+![The Grafana dashboard](/images/docs/grafana/dashboard.png)
+
+Each panel in the dashboard tracks a different metric that can be used to understand some aspect of how a Geth node is behaving. There are three main categories of panel in the default dashboard: System, Network and Blockchain. The individual panels are explained in the following sections.
+
+## What do the panels show?
+
+### System
+
+Panels in the System category track the impact of Geth on the local machine, including memory and CPU usage.
+
+#### CPU
+
+![The CPU panel](/images/docs/grafana/cpu.png)
+
+The CPU panel shows how much CPU is being used as a percentage of one processing core (i.e. 100% means complete usage of one processing core, 200% means complete usage of two processing cores). There are three processes plotted on the figure. The total CPU usage by the entire system is plotted as `system`; the percentage of time that the CPUs are idle waiting for disk i/o operations is plotted as `iowait`; the CPU usage by the Geth process is plotted as `geth`.
+
+#### Memory
+
+![The Memory panel](/images/docs/grafana/memory.png)
+
+Memory tracks the amount of RAM being used by Geth. Three metrics are plotted: the cache size, i.e. the total RAM reserved for Geth (default 1024 MB) is plotted as `held`; the amount of the cache actually being used by Geth is plotted as `used`; the memory allocations being made is plotted as `alloc`. 
```

Comment:
`the memory allocations being made is plotted as alloc.` I have a nitpick here but it's ok to ignore. 

The alloc actually represents the number of bytes allocated by system per second. Maybe we can improve the sentence a bit?

Reply:
```suggestion
Memory tracks the amount of RAM being used by Geth. Three metrics are plotted: the cache size, i.e. the total RAM reserved for Geth (default 1024 MB) is plotted as `held`; the amount of the cache actually being used by Geth is plotted as `used`; the number of bytes being allocated by the system per second is plotted as `alloc`. 
```